README.md

```diff
@@ -212,6 +212,8 @@ lib/linguist.rb
 …
 ```
 
+If you are getting permission errors ("repository path is not owned by current user"), specify your current UID and GID with `-u` option: `-u$(id -u):$(id -g)`.
```

Comment:
This is only peripherally relevant to Linguist, addresses a very specific (and easily diagnosed) scenario, and is arguably something that [Docker's own documentation](https://docs.docker.com/engine/reference/commandline/run/) does a better job of explaining (concerning permission errors raised by `docker run`).

Reply:
but it's a linguist  who throws this error, not the docker. how can it be "peripherally relevant"?

Reply:
@delameter Linguist doesn't have a `-u` switch, and the command that you're _actually_ passing `-u$(id -u):$(id -g)` to is [`docker run --rm -v`](https://github.com/github-linguist/linguist/blob/e855ef2b6f90c34074061a2e17acbe853e61b483/README.md?plain=1#L196) (which in turn runs Linguist, albeit in a controlled environment with permissions that vary depending on what options `docker run` has been executed with.